### DISCUSSION TITLE
Proposed DEI Initiative: Contributor mentoring program

### DISCUSSION THREAD

**Discussion by AMZN-Liv** [o3de/community#73] [Diversity]
Past research has shown that [gender bias exists in open source projects](https://phys.org/news/2017-05-gender-bias-open-source.html) when gender minority contributors are not already part of a project's in-group/maintainer base. As one way to try addressing this bias, O3DE could implement a leadership and maintainer mentoring program managed by the D&I committee with the goal of creating and running an onboarding program to help accelerate the growth of a diverse contributor community.

**Comment by spotz** [comment]
I will say we've run these before in other projects and a lot will depend on activity and motivation. For whatever reason, internal project mentoring (ie. new contributor experienced contributor) doesn't seem to work as well as the more external ones like GSoC and Outreachy where it's someone totally external to the project joining specifically for the mentoring opportunity.

**Reply by sptramer** [reply]
If this is the case should we be looking for local user groups around where O3DE community are already located? We discussed this a bit in the marketing meeting but marketing/DEI (or both?) might find a master list of volunteers / locations / groups useful.

**Reply by spotz** [reply]
I think doing things at a live event will help at least at that point in time, think Speed Mentoring at events which is usually packed. We have tried both one-on-one and pods(one or 2 mentors with multiple mentees to share the load) and not much long term interaction. I think the external programs work for a few reasons, getting work experience and potentially a job, but also some have stipends.
